frappe/public/scss/desk/list.scss

```diff
@@ -81,6 +81,29 @@
 	@include get_textstyle("base", "regular");
 	border-radius: var(--border-radius-md);
 
+	@media (max-width: 768px) {
```

Comment:
```suggestion
	@media (max-width: map-get($grid-breakpoints, "md")) {
```

Reply:
This doesn't work for me some reason. Does it work for you?